lib/utilities/writeContinue.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * writeContinue - Handles sending a HTTP/1.1 100 Continue message to the
+ * client, indicating that the request body should be sent.
+ * @param {object} req - http request object
+ * @param {object} res - http response object
+ * @param {string} apiMethod - The API method to call
+ * @return {undefined}
+ */
+export default function writeContinue(req, res, apiMethod) {
+    const { headers } = req;
+    // The Expect field-value is case-insensitive. Thus, just check for the
+    // Expect header's presence and a valid field-value.
+    if (headers.expect && headers.expect.toLowerCase() === '100-continue') {
+        if (apiMethod) {
+            if (apiMethod === 'objectCopy' ||
+                apiMethod === 'objectPutCopyPart' ||
```

Comment:
why are objectCopy and objectPutCopyPart included here?  Is this meant for the streaming v4 cases?  You would only have streaming v4 on objectPut and objectPutPart.  There is no body in copy requests.